stats.lua

```diff
@@ -107,10 +107,15 @@ function add_video(s)
     end
 
     if append_property(s, sec, "video-codec", {prefix="Video:", nl="", indent=""}) then
-        append_property(s, sec, "hwdec-active",
+        if not append_property(s, sec, "hwdec-current",
+                        {prefix="(hwdec:", nl="", indent=" ",
+                         no_prefix_markup=true, suffix=")"},
+                        {no=true}) then
```

Comment:
Can you change this to

    {no=true, [""]=true}) then

The property returns empty string when used in a build where it doesn't exist. :/